warehouse/admin/templates/admin/projects/detail.html

```diff
@@ -108,4 +108,29 @@ <h3 class="project-name text-center">{{ project.name }}</h3>
         </div>
       </div>
   </div>
+  <div class="box box-secondary">
+    <div class="box-header with-border">
+      <h3 class="box-title">Blacklist Project</h3>
+    </div>
+
+    <form method="GET" action="{{ request.route_path('admin.blacklist.add') }}">
+      <div class="box-body">
+        <div class="form-group col-sm-4">
+          <label for="blacklistedProject">Project Name</label>
+          <input name="project" class="form-control" id="blacklistedProject" readonly="readonly" value="{{ project.normalized_name }}" placeholder="{{ project.name }}">
```

Comment:
Maybe just ditch this field all together and make it a hidden field? We should already know what project we're blacklisting since it's all over the page and we're going to have to type the name to confirm it anyways. That would narrow this form down to just comment + the submit button.

Reply:
👍